pagopa/io-app
# feat(IT Wallet): [SIW-1574] Add wallet instance revocation

## Short description
This PR adds the feature to revoke the current wallet instance in-app.

## List of changes proposed in this pull request
- Created a new route with `ItwLifecycleWalletRevocationScreen`
- Added a new state to the eID issuance machine to handle revocation
- Added the revoke button to the eID info bottom sheet
- Updated `io-react-native-wallet`

## How to test
Press the button to revoke the wallet instance and check the network: you should see a HTTP call to `/wallet/wallet-instances/current/status`.
Then check the Redux store to see the `features.itWallet` slice reset.

https://github.com/user-attachments/assets/89fb6b8b-a174-4101-a0d7-90f0b576112d
